jekyll/_cci2_ja/gitlab-integration.adoc

```diff
@@ -18,40 +19,108 @@ contentTags:
 
 このドキュメントでは、GitLab プロジェクトと CircleCI の連携について説明し、 CI/CD パイプラインを管理するための新しいコンセプトや方法を紹介します。 また、リリース予定の機能の概要もご紹介します。
 
-[#step-one-sign-up]
-== 1. ユーザー登録
+[#sign-up]
+== ユーザー登録
 
-GitLab との連携は、新規および既存の CircleCI ユーザーに行っていただけます。 link:/docs/first-steps#gitlab-signup[CircleCI のユーザー登録] のページに記載されている手順に従ってください。
+GitLab との連携は、新規および既存の CircleCI ユーザーに行っていただけます。 xref:first-steps#gitlab-signup[CircleCI のユーザー登録] のページに記載されている手順に従ってください。
 
-* GitLab アカウントを紐付けて組織を作成します。
-* プロジェクトを作成し、GitLab アカウントからリポジトリに接続します。
+NOTE: 設定するレポジトリへの API アクセスと書き込みアクセス権が必要です。 GitLab では、これは "maintainer" 以上のロールです。
 
-リポジトリと CircleCI プロジェクトを接続すると、裏では CircleCI がお客様の GitLab リポジトリ内で Webhook を登録します。 これは、プロジェクトの作成に成功後、リポジトリの **Settings > Webhooks** のページに移動して確認することができます。
+[.tab.signup.GitLab_SaaS]
+--
+When you create a new organization and connect your GitLab.com account, you will be prompted to create a new project from a repository. You may select a repo in which you have already created a `.circleci` directory at the root of the repo, with a `config.yml` file in that directory.
```

Comment:
@kbatuigas We can see in this PR what needs translating for the GL cloud ticket we were talking about earlier, so I do think it's fine that we don't catch everything at the time that we're updating the English pages, we can just look here periodically and make tickets for specific pages as an when. Let me know what you think/if that makes sense.